docs/source/bias_reduction.ipynb

```diff
@@ -11,66 +11,86 @@
    "cell_type": "markdown",
```

Comment:
<p>So you get the same results but I did it methodologically wrong. Thanks for checking. Your revision now reads much smoother</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/434/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>